src/gf-platformid

```diff
@@ -7,28 +7,28 @@ dn=$(dirname "$0")
 # shellcheck source=src/libguestfish.sh
 . "${dn}"/libguestfish.sh
 
-# Usage: gf-oemid <input image> <output image> OEMID
-# Example: gf-oemid fedora-coreos.qcow2 fedora-coreos-aws.qcow2 ec2
+# Usage: gf-platformid <input image> <output image> PLATFORMID
+# Example: gf-platformid fedora-coreos.qcow2 fedora-coreos-aws.qcow2 ec2
 #
-# This will add the coreos.oem.id=ec2 to the bootloader arguments. Intended to
+# This will add the coreos.platform.id=ec2 to the bootloader arguments. Intended to
```

Comment:
Instead of `coreos.platform.id` we should be using `ignition.platform.id` here - sorry I don't know if that bit was clear in my original issue I opened.

Reply:
check out the latest commit, cannot find coreos.oem(platform).id elsewhere within the assembler